src/main/java/frc/robot/subsystems/drivetrain/BaseSwerveSubsystem.java

```diff
@@ -419,4 +436,24 @@ public void setVisionEnabled(boolean visionEnable) {
         this.VISION_ENABLE = visionEnable;
         visionEnableEntry.setBoolean(VISION_ENABLE);
     }
+
+    public double minTargetDistance(List<PhotonTrackedTarget> trackedTargets){
+        Transform3d targetTransform = trackedTargets.get(0).getAlternateCameraToTarget();
+        double distance = Math.sqrt(Math.pow(targetTransform.getX(), 2) + Math.pow(targetTransform.getY(), 2));
+        double minAprilTagDist = distance;
+        
+        for(int i = 1; i < trackedTargets.size(); i++){
+            targetTransform = trackedTargets.get(i).getAlternateCameraToTarget();
+            distance = Math.sqrt(Math.pow(targetTransform.getX(), 2) + Math.pow(targetTransform.getY(), 2));
+            if(minAprilTagDist < distance){
+                minAprilTagDist = distance;
```

Comment:
If you wanted to get fancy with this method you can use `Collections.min()` with a custom comparator that mapped tracked targets to their `distance`s instead of writing a custom loop. You're already operating on a `List<T>`, so you wouldn't even need to convert an array to a `Collection<T>` for the operation.